src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,25 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../App';
+import { User } from '../../App';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo, index) => {
+        const { title, userId } = todo;
+        const currentUser: User = (usersFromServer || []).find(
```

Comment:
Ensure that `usersFromServer` is always defined and is an array. If it's undefined, the fallback to an empty array will prevent errors, but it will also mean that no users are found, which might not be the intended behavior.